core/src/test/java/com/klaytn/caver/common/ContractTest.java

```diff
@@ -845,64 +847,79 @@ public void call() throws IOException, ClassNotFoundException, InvocationTargetE
         Caver caver = new Caver(Caver.DEFAULT_URL);
 
         Contract contract = new Contract(caver, jsonObj, contractAddress);
-        SendOptions sendOptions = new SendOptions(LUMAN.getAddress(), DefaultGasProvider.GAS_LIMIT);
+        CallObject callObject = CallObject.createCallObject(
+                LUMAN.getAddress(),
+                null,
+                null,
+                null,
+                null);
 
-        List<Type> result = contract.getMethod("symbol").call(null, sendOptions);
+        List<Type> result = contract.getMethod("symbol").call(null, CallObject.createCallObject());
```

Comment:
createCallObject  should be called here again? How about passing callObject to call function?